### ISSUE TITLE
Redirector maintainer

### ISSUE THREAD

**Issue by gzur** [Gitoffthelawn/Contact-Gitoffthelawn#8]
Hi, 

Einar, the maintainer of [einaregilsson/Redirector](https://github.com/einaregilsson/Redirector/) has passed away.

As a result, the Redirector is in need of a new maintainer. 

I am working on settling Einar's affairs on behalf of his widow, but I lack the expertise to maintain the Redirector.

I decided to ping you, since after Einar, you are the [number 2 contributor](https://github.com/einaregilsson/Redirector/graphs/contributors), (alongside [@gkrishnaks](https://github.com/gkrishnaks) but he doesn't appear very active).

Would you be interested in taking up the mantle?

_P.S. I really dig this repo-as-contact-form setup you've got going._

**Comment by Gitoffthelawn** [comment]
@gzur Oh my goodness. I'm stricken with grief to learn of this horrible news. Yes, I'll definitely help. Einar was so young and full of life. I'm going to miss him, and I'm sure others will too. Please express heartfelt condolences on behalf of myself and the GitHub community to his family. I have a lump in my throat and my stomach is in knots. I'll write back again after I process this a little.

**Comment by Gitoffthelawn** [comment]
@gzur It took me a few days to begin processing this loss. It caught me by surprise, especially given Einar's young age. There are a few people on GitHub that will always hold a special place in my heart, and Einar is one of them.

Would his family benefit from a fundraiser and be open to it?  Perhaps the sizable Redirector userbase could be called to action.  Of course, I'm thinking only if it would be welcomed by his loved ones... it's just an idea I had while thinking about Einar's passing (and I try not to cling to ideas).

Regarding Redirector, anytime you want to talk about that project (and perhaps a couple others), I'm open to it.  No rush.  I keep pretty busy, but Einar and that project have special meaning to me, so I'll do my best to make time for what's needed.

**Comment by gzur** [comment]
Sorry for keeping you waiting.

> Would his family benefit from a fundraise

There is no need for a fundraiser, but thank you for your kind thoughts.

---
Regarding Redirector, I have no insight into the project, but would be willing to assist with ensuring its continued existence. 

I'm not familiar with the release/build process and whether they rely on any existing accounts (inside the Chrome store et.al.) - but I might be able to recover them if we need them.

What the Redirector would need, if I'm not mistaken, is somebody who:
* Can have opinions on the incoming PR's / Issues
* Has the inclination and interest for the above

Is that a role you can imagine yourself assuming?

We will probably end up with forking the repository and putting up a notice in [einaregilsson/Redirector](https://github.com/einaregilsson/Redirector)

**Comment by Gitoffthelawn** [comment]
You're welcome.

Every time I see Einar's name, I still get a little lump in my throat.  But I'm also starting to feel some joy from simply remembering him.  Grieving is an interesting process, and always seems to be unique.

The roles you mentioned regarding Redirector sound good to me.  I consider it an essential extension and would like to help keep it going.

Any help you (and others) can offer, especially on the Chrome side, will be most appreciated.

Regarding forking or using the same repository, either is good by me.  The advantage to keeping the existing repository is that nothing needs to change.  The advantage to forking is that if Einar had anything private in his account, neither of us will ever see it, which is the respectful thing to do.  Of course, if there is nothing private in the account, then I don't think it really matters.

If I may ask, what is/was (I never know which tense to use when someone passes) your relationship with Einar?

**Comment by gzur** [comment]
Hi, sorry for dropping off the radar. Life got in the way.

For now, I'll make you a collaborator of the repository.

We can see where it goes from there

**Comment by Gitoffthelawn** [comment]
> Hi, sorry for dropping off the radar. Life got in the way.
> 
> For now, I'll make you a collaborator of the repository.
> 
> We can see where it goes from there

No problem.  I completely understand.  I received the invite, and accepted it.  Thanks.

I'll try a small change (to the README,md) to see if everything works as expected.

**Comment by Gitoffthelawn** [comment]
> Hi, sorry for dropping off the radar. Life got in the way.
> 
> For now, I'll make you a collaborator of the repository.
> 
> We can see where it goes from there

The community is requesting some updates, and I'm ready to build them and publish.

If you will, can you please give me the appropriate access to the Redirector GitHub repo and its Mozilla and Chrome accounts?  If there is an Opera account, I can take a look at that too.

Thanks in advance.

**Comment by Gitoffthelawn** [comment]
@gzur Hi. I'm making another attempt to ping you.  I would like to release community improvements to Redirector, but I still need the appropriate access to the Redirector GitHub repo to do so, as well as Redirector's Mozilla and Chrome accounts (and Opera, if it's available).  Can you arrange this, please?

**Comment by Gitoffthelawn** [comment]
@gzur People are eager for a new Redirector release, and I would like to facilitate that happening.  If you can provide the requisite access, please, I can make it happen.

**Comment by Gitoffthelawn** [comment]
@gzur Any update?

**Comment by gzur** [comment]
Hi @Gitoffthelawn 

_Sorry for dropping off the radar (again)._

Regrettably, I do not have access to the store accounts you mention, so I think it would be best to create new accounts.

I have also made you admin of the repo.

If you need to reach me for anything, it would be best to just email me at 
```
gzur [at] gzur [dot] net
```

**Comment by Gitoffthelawn** [comment]
@Torro Yes, help is needed.  Thank you for your generous offer. Can you briefly summarize your skills?

**Comment by Torro** [comment]
It's my pleasure.
I can do whatever's needed, though I prefer working with code.

For more information please send me an email: k5ap349d8@mozmail.com.

**Comment by Gitoffthelawn** [comment]
@Torro Sounds great.  Thanks again.  I'll be in touch via email as soon as I have some free time.  I have a full schedule the next several days, but my schedule will (hopefully) open up a bit afterwards.  Your timing is perfect, as I was hoping to work on Redirector soon.

**Comment by Gitoffthelawn** [comment]
@Torro Haven't forgotten about you... just super busy.

**Comment by stdedos** [comment]
I can be added as a "backup contact", the project does not have enough people to ensure the legacy continues.

I don't have extension developing experience - but I write the errant js/ts
